packages/react/src/CTABanner/CTABanner.tsx

```diff
@@ -0,0 +1,99 @@
+import React, {forwardRef, type Ref} from 'react'
+import clsx from 'clsx'
+import type {BaseProps} from '../component-helpers'
+import {Heading, HeadingTags} from '../Heading'
+import {Text} from '../Text'
+import {ButtonGroup} from '../ButtonGroup'
+
+/**
+ * Design tokens
+ */
+import '@primer/brand-primitives/lib/design-tokens/css/tokens/functional/components/cta-banner/colors-with-modes.css'
+
+/** * Main Stylesheet (as a CSS Module) */
+import styles from './CTABanner.module.css'
+
+export type CTABannerProps = BaseProps<HTMLDivElement> & {
+  children: React.ReactNode | React.ReactNode[]
```

Comment:
Can we add JSDoc annotations to these props? [Example from SubdomainNavBar](https://github.com/primer/brand/blob/f41b6cf4c020e67dc94cce8c6816d68ebaeb7b75/packages/react/src/SubdomainNavBar/SubdomainNavBar.tsx#L17). 

These are really useful for app devs, as IDE's like VSCode will forward them to all their references on hover. I will usually reuse the description in the docs prop table too, so it's helpful in other ways also.

Reply:
Sounds good to me!